phpstan.neon

```diff
@@ -1,4 +1,5 @@
 parameters:
+    phpVersion: 80000
```

Comment:
Are we sure setting this is a wise choice?

Reply:
I've added this because `curl_init()` returns a `CurlHandle` object in PHP8 instead of a resource, so the result of PHPStan was different depending on the PHP version used to run it.

I don't know if it would work but we could modify `build/phpstan/bootstrap.php` by adding something like that:
```php
if (!class_exists('CurlHandle')) {
    class CurlHandle {}
}
```

Reply:
I admit I didn't look into that in depth, but the `CurlHandle` return type should break things for PHP 7.x then?

Reply:
With this configuration PHPStan acts like it is run with PHP 8.0 even if another version is actually used, so it won't fail with PHP 7.x.

Reply:
That's not what I meant: For the time being, `phive` needs to run under older Versions of PHP down to PHP 7.3. If the code requires a `CurlHandle` instance to be returned but that's PHP 8+ only, the code in question won't run in PHP 7.x.

That would be a problem.